docs/rum_collection/advanced_configuration.md

```diff
@@ -117,10 +117,23 @@ For more details and available options, refer to the code documentation comments
 
 ## Track custom global attributes
 
-In addition to the [default RUM attributes][11] captured by the mobile SDK automatically, you can choose to add additional contextual information, such as custom attributes, to your RUM events to enrich your observability within Datadog. Custom attributes allow you to slice and dice information about observed user behavior (such as cart value, merchant tier, or ad campaign) with code-level information (such as backend services, session timeline, error logs, and network health).
+In addition to the [default RUM attributes][11] captured by the mobile SDK automatically, you can choose to add additional contextual information, such as custom attributes, to your RUM events to enrich your observability within Datadog. 
+
+Custom attributes allow you to slice and dice information about observed user behavior (such as cart value, merchant tier, or ad campaign) with code-level information (such as backend services, session timeline, error logs, and network health).
+
+### Set a custom global attribute
+
+To set a custom global attribute, use `Global.rum.addAttribute()`.
+
+* To add an attribute, set `rumMonitor.setAttribute(forKey: "some key", value: "some value")`.
+* To update the value, set `rumMonitor.setAttribute(forKey: "some key", value: "some other value")`.
+* To remove the key, set `rumMonitor.setAttribute(forKey: "some key", value: nil)`.
+* To return global attributes and add to events, set `let globalAttrs = rumMonitor.attributes`.
```

Comment:
```suggestion
* To add an attribute, use `Global.rum.setAttribute(forKey: "some key", value: "some value")`.
* To update the value, use `Global.rum.setAttribute(forKey: "some key", value: "some other value")`.
* To remove the key, use `Global.rum.removeAttribute(forKey: "some key")`.
```
Because `Global.rum` is served from SDK, whereas `rumMonitor` would be some custom variable made by the user.

Also, attributes are removed with `removeAttribute(forKey:)` and we have no way to return attributes (think: users know all attributes they set).